File: src/main/java/com/tn/service/EmpService.java
package com.tn.service;


import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.tn.Entity.Employee;
import com.tn.repository.employee;

@Service
public class EmpService {

	
	@Autowired
	employee Emp;
	
	
	public String createEmp(Employee emp) {
		
		Emp.save(emp);
		return "create successfully";
		
	}
	
	public List<Employee> getAll(){
		
		return Emp.findAll();
	}
	
	public boolean deleteEmp(Long id) {
		
		Emp.deleteById(id);
		return true;
	}
	
	public String UpdateEmployee(Long id , Employee employee) {
		
		Employee newemp = Emp.findById(id).get();
		newemp.setName(employee.getName());
		newemp.setPhone(employee.getPhone());
		newemp.setEmail(employee.getEmail());
		
		Emp.save(newemp);
		return "Update Successfully";
	}

	 

	public Employee getEmployeeById(Long id) {
		 Employee emp = Emp.findById(id).get();
		return emp;
	}
	
	
	
}

File: src/Service/EmpService.js

import axios from 'axios';

const EMPLOYEE_API_BASE_URL= "http://localhost:7070"
class EmpService{
    saveEmployee(employee){
       return axios.post(EMPLOYEE_API_BASE_URL+"/createEmp" ,employee)
    }

    getEmp(){
        return axios.get(EMPLOYEE_API_BASE_URL+"/GetAll")
    }

   deleteById(id){
    return axios.delete(EMPLOYEE_API_BASE_URL + "/deleteEmp/" + id);
   }

    UpdateEmp(employee , id){
        return axios.put(EMPLOYEE_API_BASE_URL+ "/employee/"+id , employee)
     }

     getEmployeeById(id){
        return axios.get(EMPLOYEE_API_BASE_URL+ "/GetEmp/" + id)
     }
}

export default new EmpService(); 